internal/resolution/pm/gomod/job_test.go

```diff
@@ -19,16 +19,66 @@ func TestNewJob(t *testing.T) {
 }
 
 func TestRunGraphCmdErr(t *testing.T) {
-	cmdErr := errors.New("cmd-error")
-	cmdFactoryMock := testdata.NewEchoCmdFactory()
-	cmdFactoryMock.MakeGraphCmdErr = cmdErr
-	j := NewJob("file", cmdFactoryMock, nil)
-
-	go jobTestdata.WaitStatus(j)
-
-	j.Run()
-
-	assert.Contains(t, j.Errors().GetCriticalErrors(), util.NewPMJobError(cmdErr.Error()))
+	cases := []struct {
+		name  string
+		error string
+		doc   string
+	}{
+		{
+			name:  "General error",
+			error: "cmd-error",
+			doc:   util.UnknownError,
+		},
+		{
+			name:  "Invalid package version",
+			error: "        |go: errors parsing go.mod:\n        |/home/asus/Projects/playground/imessage/go.mod:8:2: require github.com/google/uuid: version \"v111.5.0\" invalid: should be v0 or v1, not v111",
+			doc:   "Failed to find package github.com/google/uuid. Please check that package versions are correct in the manifest file. It should be v0 or v1, not v111.",
```

Comment:
```suggestion
			doc:   "Failed to find package \"github.com/google/uuid\". Please check that package versions are correct in the manifest file. It should be v0 or v1, not v111.",
```
If change is made for dependency name this test will need to change as well